src/main.lua

```diff
@@ -370,6 +370,19 @@ end, function(msg)
 		lastKnownRequestSupply = LastKnownPnpRequestSupply,
 		lastKnownTotalSupply = lastKnownTotalTokenSupply(),
 	}
+
+	-- Format all incoming addresses
+	msg.From = utils.formatAddress(msg.From)
+	msg.Tags.Recipient = msg.Tags.Recipient and utils.formatAddress(msg.Tags.Recipient) or nil
```

Comment:
Nit: this could be done in a for loop over an array of known tags.

Reply:
[a4ad6cd](https://github.com/ar-io/ar-io-network-process/pull/183/commits/a4ad6cdd20fbc2816860b332288243d6cf78d500)